imod/mf6/riv.py

```diff
@@ -335,9 +336,14 @@ def from_imod5_data(
         optional_river_package: Optional[River] = None
         optional_drainage_package: Optional[Drainage] = None
         # create a drainage package with the conductance we computed from the infiltration factor
+        drainage_arrays = {
+            "stage": regridded_package_data["stage"],
+            "conductance": drain_conductance,
+        }
+        drainage_arrays = mask_arrays(drainage_arrays)
```

Comment:
This is still some leftover behind-the-scenes cleaning, which we need to avoid. Could you revert this as well?

Reply:
done